docs/content/en/blog/articles/minio-backup/index.md

```diff
@@ -0,0 +1,268 @@
+```
+
+You can then deploy K8ssandra with the following custom values file (all default values will be used if not customized here) :
+
+```
+medusa:
+  enabled: true
+  storage: s3_compatible
+  storage_properties:
+      host: minio.minio.svc.cluster.local
+      port: 9000
+      secure: "False"
+  bucketName: k8ssandra-medusa
+  storageSecret: medusa-bucket-key
+```
+
+Save the above file as k8ssandra_medusa_minio.yaml and then install K8ssandra with the following command:
+
+```
+helm install k8ssandra k8ssandra/k8ssandra -f k8ssandra_medusa_minio.yaml -n k8ssandra
+```
+
+Now wait for the Cassandra cluster to be ready by using the following wait command:
+
+```
+kubectl wait --for=condition=Ready cassandradatacenter/dc1 --timeout=900s -n k8ssandra
+```
+
+You should now see a list of pods similar to this:
+
+```
+% kubectl get pods -n k8ssandra
+NAME                                                  READY   STATUS      RESTARTS   AGE
+k8ssandra-cass-operator-547845459-dwg68               1/1     Running     0          6m36s
+k8ssandra-dc1-default-sts-0                           3/3     Running     0          5m56s
+k8ssandra-dc1-stargate-776f88f945-p9twg               0/1     Running     0          6m36s
+k8ssandra-grafana-75b9cb64cc-kndtc                    2/2     Running     0          6m36s
+k8ssandra-kube-prometheus-operator-5bdd97c666-qz5vv   1/1     Running     0          6m36s
+k8ssandra-medusa-operator-d766d5b66-wjt7j             1/1     Running     0          6m36s
+k8ssandra-reaper-5f9bbfc989-j59xk                     1/1     Running     0          2m48s
+k8ssandra-reaper-operator-858cd89bdd-7gfjj            1/1     Running     0          6m36s
+k8ssandra-reaper-schema-4gshj                         0/1     Completed   0          3m3s
+prometheus-k8ssandra-kube-prometheus-prometheus-0     2/2     Running     1          6m32s
+```
+
+# Create some data and back it up
+
+Next, create some data in Cassandra by creating a **test_data.cql** file:
+
+```
+CREATE KEYSPACE medusa_test  WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1};
+USE medusa_test;
+CREATE TABLE users (email text primary key, name text, state text);
+insert into users (email, name, state) values ('alice@example.com', 'Alice Smith', 'TX');
+insert into users (email, name, state) values ('bob@example.com', 'Bob Jones', 'VA');
+insert into users (email, name, state) values ('carol@example.com', 'Carol Jackson', 'CA');
+insert into users (email, name, state) values ('david@example.com', 'David Yang', 'NV');
+```
+
+And copy it into the Cassandra pod (the StatefulSet one, which contains <code>-<strong>sts</strong>-</code> in its name):
+
+```
+kubectl cp test_data.cql k8ssandra-dc1-default-sts-0:/tmp -n k8ssandra -c cassandra
+```
+
+Now extract the password to access Cassandra with the k8ssandra-superuser (the password is different for each installation unless it is explicitly set at install time): 
```

Comment:
Do we want to pre-set the password for the cluster so it's a known value?

Reply:
I'd say it's good for folks to be aware of this. wdyt?

Reply:
This works too, I'm open to either.